src/routes/user-event.ts

```diff
@@ -0,0 +1,100 @@
+import asyncHandler from 'express-async-handler';
+import httpErrors from 'http-errors';
+import { Router } from 'express';
+import { getUserFromRequest } from '../lib/AuthHelper';
+import { addToCollection, DB_PATHS, updateDocument } from '../lib/DBHelper';
+import { sanitizeString } from '../lib/DataValidator';
+import { getDb } from '../lib/Firebase';
+import { EVENT_STATUS } from '../lib/EventHelper';
+
+const router = Router();
+
+/**
+ * Update event users
+ * Sample JSON POST
+ {
+    "eid": "8UhyXDgfoBAJBboJkMQQ",
+    "uid": "42fdsGW3teCAwwoJkMQQ"
```

Comment:
uid won't be required in the body, you're getting the user from the `const user = await getUserFromRequest(req);` Only need the eid and status here